ios/RNCallKeep/RNCallKeep.m

```diff
@@ -862,7 +862,12 @@ + (CXProviderConfiguration *)getProviderConfiguration:(NSDictionary*)settings
     providerConfiguration.supportsVideo = YES;
     providerConfiguration.maximumCallGroups = 3;
     providerConfiguration.maximumCallsPerCallGroup = 1;
-    providerConfiguration.supportedHandleTypes = [RNCallKeep getSupportedHandleTypes:settings[@"handleType"]];
```

Comment:
Why don't you pass `generic` to `handleType` as settings in your javascript code ?

Reply:
when I met this issue, every handleType work as expected except generic type. I think when press generic type at native call log, it not pass any type to handleType so the start call action not be trigger

Reply:
I don't get it, the `[RNCallKeep getHandleType:settings[@"handleType"]]` should return `CXHandleTypeGeneric` if the `_handleType` is `generic`.
So if you passe `generic` it will work as expected.